.github/workflows/publish.yml

```diff
@@ -48,7 +47,7 @@ jobs:
           yarn install --non-interactive --frozen-lockfile --cache-folder ${{
           steps.yarn-cache-dir-path.outputs.dir }}
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Did you set this? I tried a prerelease, but it doesn't seem to work: https://github.com/uplift-ltd/nexus/actions/runs/3466437410/jobs/5790333619

I see the env var in actions, so I'm not sure what's up https://github.com/uplift-ltd/nexus/settings/secrets/actions